apollo-client/src/main/java/com/ctrip/framework/apollo/util/yaml/YamlParser.java

```diff
@@ -182,24 +185,6 @@ private void buildFlattenedMap(Map<String, Object> result, Map<String, Object> s
       }
     }
 
-    @Override
```

Comment:
We may also remove the StrictMapAppenderConstructor class?

Reply:
fixed